app/services/form_to_resource_service.rb

```diff
@@ -28,6 +25,10 @@ def convert(params, work)
       # Process contributors
       resource = process_contributors(params, resource)
 
+      # Process funders
+      # (New pattern: method modifies resource in place.)
+      process_funders(params, resource)
```

Comment:
I would prefer if the method returns the updated resource,  Just to make the code clear that the resource is being changed by the method.  I think that is the pattern process_contributors takes.  That said we should make it all constant.

Reply:
My preference would be that it either modify in place, or assign to a new instance. Right now it's doing both, and I think that could be confusing. `process` is pretty vague, so I'll rename the methods for clarity.